src/main/sensors/battery.c

```diff
@@ -426,6 +426,14 @@ float calculateVbatPidCompensation(void) {
     return batteryScaler;
 }
 
+float calculateVbatThrottleCompensation(void) {
+    float batteryScaler =  1.0f;
+    if (batteryConfig()->voltageMeterSource != VOLTAGE_METER_NONE && batteryCellCount > 0) {
+        batteryScaler =  constrainf((( (float)batteryConfig()->vbatmaxcellvoltage * batteryCellCount ) / (float) voltageMeter.filtered), 1.0f, 1.6f);
```

Comment:
I'm not sure about the hardcoded 1.6x max boost.  Perhaps this should be tuneable as a configuration parameter.  160% is quite a maximum boost and may not be suitable to all applications.  Also the only way to tune this currently would be to adjust the minimum cell voltage but this would also affect the vbat PID comp.

Reply:
Also using max cell voltage means that compensation will be always larger that one. Maybe using reference voltage for 100% throttle would work better